tools/configure_server_settings.rb

```diff
@@ -64,43 +65,45 @@ def types_valid?(old_val, new_val)
   end
 end
 
-server = MiqServer.where(:id => opts[:serverid]).take
-unless server
+server_list = opts[:all_servers] ? MiqServer.all : MiqServer.where(:id => opts[:serverid])
+unless server_list
```

Comment:
shouldn't this be `unless server_list.empty?`

Reply:
`unless server_list.empty?` is the equivalent of `if server_list.empty? == false` 
if the server list is not empty, then we do not want to run line 70, so I don't believe so.

Reply:
I think @chessbyte is more asking if there needs to be something additional, because server_list will always have a value and this code will never execute as is (at worst, server_list will be an empty array which is always truthy)

Reply:
True, if we care about the puts and early exit on line 70-71, we should check empty.